sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -631,6 +631,17 @@ frameBound
     | expression boundType=(PRECEDING | FOLLOWING)
     ;
 
+excludeClause
+    : EXCLUDE excludeType=CURRENT ROW
+    | EXCLUDE excludeType=GROUP
+    | EXCLUDE excludeType=TIES
+    | EXCLUDE excludeType=NO OTHERS
+    ;
+
+explainOption
```

Comment:
hmm. I don't think I introduced this change.. maybe I need to rebase to clean up my code before meaningful review. Thanks!